src/components/PersonLink.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type PropsPerson = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<PropsPerson> = ({ person }) => {
+  return (
+    <Link
+      to={`/people/${person.slug}`}
+      className={cn({ 'has-text-danger': person.sex === 'f' })}
+    >
+      {person.name}
```

Comment:
Use [destruction assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to a person object